vc_vidyo/indico_vc_vidyo/api/client.py

```diff
@@ -79,9 +79,28 @@ def create_room_object(self, **kwargs):
     def find_room(self, extension):
         filter_ = self.client.factory.create('Filter')
         filter_.query = extension
+        filter_.limit = 40
         filter_.dir = 'DESC'
 
-        return self.soap.getRooms(filter_).room
+        target_room = None
+        counter = 0
+        from indico_vc_vidyo.plugin import VidyoPlugin
```

Comment:
We group imports at the top of the file unless they cause a circular import. In that case, we put them as the first lines of the function.